Sort tags as versions

ddramowicz opened the pull request:
The iOS devs noticed the beta changelog has not really been working since we've moved to versions 7.10 and 7.11. This could be because the script is not sorting the tags the right way.
Apparently it's possible to use `version:refname` to have the tags treated as versions when sorting. 
If this is correct we should also apply it to the app store changelog (Or whatever the correct fix would be)

ddramowicz committed:
Sort as versions

jorgeazevedo commented:
ah! super : -D I really wasn't sure how to fix that, thanks so much for looking into it!

There's a few other bits to fix before the algorithm will work again I think, do you mind if I push some commits to this branch @ddramowicz ?

jorgeazevedo committed:
Include 7.XX tags as well as 7.X

Careful! That git tag argument is _not_ a regex, it's unix file
pattern matching. You can see the docs here
https://docs.python.org/2/library/fnmatch.html

jorgeazevedo committed:
Apply version: prefix to release changelog as well

jorgeazevedo reviewed (commented):
Thanks for your help, that change to the sort argument was spot on!
